MaxText/pyconfig.py

```diff
@@ -660,7 +658,6 @@ def pipeline_first_axis(raw_keys):
           "fsdp",
           "fsdp_transpose",
           "sequence",
-          "context",
```

Comment:
Why is it removed from mesh_axes and data_sharding?

Reply:
this PR is to completely remove `context ` axis introduced by inference seq sharding and rename it to `context_autoregressive'. I will add back `context_autoregressive` here